docker-compose.yml

```diff
@@ -3,8 +3,8 @@ version: '3.8'
 services:
   jekyll-host:
     image: container.ldtteam.com/ldtteam/minecolonies/wiki:latest
-    volumes:
-      - .:/srv/jekyll
+    ports:
```

Comment:
Don't expose ports in the compose.
This is a runtime security risk and not needed.

If you need to modify the port on which our cluster ingress access the container, you need to modify the labels attached to the container.